qdmr_parsing/model/run_model.py

```diff
@@ -4,12 +4,19 @@
 import os
 import pandas as pd
 import random
+import sys
+from copy import deepcopy
 
+sys.path.append('.')
```

Comment:
Wasn't able to import without `sys.path.append`

Reply:
This should not be solved in the code, but by adding the root directory, which is `qdmr_parsing`, to the python path in your environment. See #1.